src/Routes.js

```diff
@@ -25,12 +25,8 @@ InsightsRoute.propTypes = {
 export const Routes = () => {
     return (
         <Switch>
-            { window.insights.chrome.isBeta() ?
-                <InsightsRoute exact path='/baselines' component={ BaselinesPage } /> : null
-            }
-            { window.insights.chrome.isBeta() ?
-                <InsightsRoute path='/baselines/:id' component={ EditBaseline } /> : null
-            }
+            <InsightsRoute exact path='/baselines' component={ BaselinesPage } /> : null
```

Comment:
looks like you have both sides of the if/else here "`: null`"

Reply:
You are right, fixed it up